### DISCUSSION TITLE
Receiving error when trying to use DLRS

### DISCUSSION THREAD

**Discussion by SKZGirl** [SFDO-Community/declarative-lookup-rollup-summaries#1488] [General]
The error is the following. I have looked in the community but have not found resolution:
Errors

Unable to connect to the Salesforce Metadata API.
A Remote Site Setting must be created in your org before you can use this tool.
Press the Create Remote Site Setting button to perform this step or refer to the post install step below to perform this manually.

When I go to create Remote Site setting I receive an error that there is already one created.

**Comment by aheber** [comment]
@SKZGirl the best thing to do is to post on the [Trailblazer Community Group](https://success.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009O5p) with requests for help.

Probably the best thing you can do is delete the existing Remote Site Setting (with `dlrs` in the name) via the Setup page. Once that is done then return to the DLRS app and try and re-add the Remote Site Setting. Often if you've gone through a pod migration or moved to Hyperforce then it can change the URL for your org and will need the Remote Site Setting to be reset.

It is possible that you have additional security protections or other considerations in place that prevent how DLRS is trying to connect to your org, if that is the case then it may get harder to use this tool.

If you post in that Trailblazer Community Group there should be more individuals with a lot more experience to help should you run into any additional trouble.
